src/app/body/task-details/log-work/log-work.component.html

```diff
@@ -17,14 +17,23 @@ <h5 class="active nav-link head" id="exampleModalLabel">Log Work</h5>
             </nav>
             <div class="container modal-body align-items-center px-4" id="logWorkPage">
                 <form *ngIf="!showClose" (ngSubmit)="submit()" #form="ngForm">
+                    <div class="row px pt-2">
```

Comment:
What is px doing here

Reply:
Actually that was part of the previous code written by someone else.

Reply:
Actually that was part of the previous code written by someone else.

Reply:
It's showing as your change